src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
1.Why don't you use descturturization? 
```suggestion
    ? people.filter(({ sex, died }) => sex === 'm' && Math.ceil(died / 100) === century)
    : people.filter((p) => p.sex === 'm');
```
Else, please, declare arguments with meaningful names. What's **p**?
2. You can use 1 filter method and move your condition into its callback